### DISCUSSION TITLE
Neper -V fails when plotting mesh

### DISCUSSION THREAD

**Discussion by damulaoshi** [neperfepx/neper#170] [Q&A] [answered]
Dear Romain,
When I want to print the meshed models using "neper -V .. -dataeltcol ori -print ..", it shows "aborted (core dumped)" (see the picture below). 
Why does this happen? How can I fixed this problem? (I guess maybe the grain size is too small and can't be printed when I use Module-V).

The code is here:
```
neper -T -n 8000 -domain "cube(1,1,1)" -o delta20
neper -M delta20.tess -order 1 -elttype hex -rcl 2 -format msh,inp
neper -V delta20.msh -dataeltcol ori -print delta20mesh
```

![image](https://user-images.githubusercontent.com/75380898/113564894-67307380-960a-11eb-8f32-a54e6e48ecf4.png)

Thank you before and hope to receive your answer.
Best regards,
Damu

**Comment by rquey** [comment, marked as answer]
This was a bug, just fixed in `4.1.3-9`.  Thanks for the report.

**Reply by damulaoshi** [reply]
No problem! Thank you!
